ksql-engine/src/main/java/io/confluent/ksql/topic/TopicDeleteInjector.java

```diff
@@ -75,6 +78,26 @@ public TopicDeleteInjector(
         Objects.requireNonNull(schemaRegistryClient, "schemaRegistryClient");
   }
 
+  private void checkTopicRefs(final DataSource<?> source) {
```

Comment:
super nit: private methods to the bottom of class files, not interspersed with the public API.